### PR TITLE
add OLS internal team rituals to the slack page

### PR DESCRIPTION
**Pull request by yochannah**
:+1::tada: First of all, thanks for taking the time to contribute! :tada::+1:

## FOR CONTRIBUTOR

* [x] I have read the [CONTRIBUTING.md](https://github.com/open-life-science/open-life-science.github.io/blob/main/CONTRIBUTING.md) document

PR Type: 
* [ ] This PR adds a new blog post
* [x] This PR does something else: Adds content to the slack page, mostly intended for OLS internal usage.

 
## FOR REVIEWERS

Thanks for taking the time to review! :heart:

Here are the list of things to make sure of:
* [ ] The website builds (a check will fail if not)
* [ ] All images have been added within the Pull Request and they have Alt text
* [ ] If there are paragraphs or text, the key messages are highlighted
* [ ] All internal links (within OLS website) use the [`{% link path_to_file.md %}` format](https://jekyllrb.com/docs/liquid/tags/#link)
* [ ] The preview corresponds to the changes described in the Pull Request
* [ ] The code is tidy and passes the linting tests

### CODE REVIEW

File: policies-procedures-and-docs/communications/slack-guide.md [hunk 1]
```diff
@@ -181,3 +181,23 @@ Yes, you can send files (< 1 GB) in public channels and direct messages. See thi
...
+
+---
+
+## Specific channels and rituals: 
+
+- `#internal-work`: synchronous and asynchronous communication. Some topics may go in a specific sub-channel, e.g. a nebula-specific internal channel, the web/design team channel, etc. 
+    - :calendar: Start-of-week ritual: Go through your emails, slacks, and to-do lists. What's the plan this week? Post a few notes in Slack so others know what's one your mind. 
+- `#internal-casual`: Post your cat and your baby here, and a picture of your cappuccino. 
+- `#internal-project-updates`:
+    -  :calendar: Twice a month ritual: Post a short update on the successes, challenges and plans or opportunities you've had in your project over the last few weeks. These are generally compiled once a month into a report and shared with the entire team and with Governance. [Report template](https://github.com/open-life-science/internal-policies-and-procedures/tree/main/ols-reports).
```
Comment (iramosp) [inline policies-procedures-and-docs/communications/slack-guide.md:202]:
```suggestion
    -  :calendar: Twice a month ritual: Post a short update on the successes, challenges and plans or opportunities you've had in your project over the last few weeks. These are generally compiled once a month into a report and shared with the entire team and with Governance. 
```
This is a private link, most people will only see a 404 error. I think the description already captures the key elements of the template.

**Review by iramosp — COMMENTED** [review]
It looks good to me. I added two small suggestions.

**Comment by yochannah** [comment]
thanks @iramosp - can you re-review and merge if it's okay?

**Review by iramosp — APPROVED** [review]
This looks good to me 🙌

**Pull request merged**